src/style.css

```diff
@@ -0,0 +1,32 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+  height: 16px;
+  width: 16px;
+}
+
+.stars--0 :nth-child(n),
+.stars--1 :nth-child(n+1),
+.stars--2 :nth-child(n+2),
+.stars--3 :nth-child(n+3),
+.stars--4 :nth-child(n+4),
+.stars--5 :nth-child(n+5) {
+  background-image: url(images/star.svg);
+}
```

Comment:
You can use one default style for all start, and then override it with specific selectors.